packages/config-loader/src/sources/FileConfigSource.ts

```diff
@@ -81,10 +86,13 @@ export class FileConfigSource implements ConfigSource {
 
   readonly #path: string;
   readonly #substitutionFunc?: SubstitutionFunc;
+  readonly #watch: boolean;
 
   private constructor(options: FileConfigSourceOptions) {
     this.#path = options.path;
     this.#substitutionFunc = options.substitutionFunc;
+    // Default: true
+    this.#watch = typeof options.watch !== 'undefined' ? options.watch : true;
```

Comment:
```suggestion
    this.#watch = options.watch ?? true;
```

Slightly more tidy version 😁